iceoryx2-ffi/cxx/tests/src/service_publish_subscribe_tests.cpp

```diff
@@ -1131,5 +1137,133 @@ TYPED_TEST(ServicePublishSubscribeTest, opening_existing_service_with_same_heade
     ASSERT_TRUE(sut_open.has_error());
     EXPECT_EQ(sut_open.error(), PublishSubscribeOpenError::IncompatibleTypes);
 }
+
+TYPED_TEST(ServicePublishSubscribeTest, PayloadTypeNameIsSetToRustPendantForFixedSizeIntegersAndBoolAndSlicesOfThem) {
+    constexpr ServiceType SERVICE_TYPE = TestFixture::TYPE;
+    const auto service_name = iox2_testing::generate_service_name();
+    auto node = NodeBuilder().create<SERVICE_TYPE>().expect("");
+    {
+        auto service = node.service_builder(service_name).template publish_subscribe<uint8_t>().create().expect("");
+        auto static_config = service.static_config();
+        ASSERT_THAT(static_config.message_type_details().payload().type_name(), StrEq("u8"));
+    }
+    {
+        auto service = node.service_builder(service_name).template publish_subscribe<uint16_t>().create().expect("");
+        auto static_config = service.static_config();
+        ASSERT_THAT(static_config.message_type_details().payload().type_name(), StrEq("u16"));
+    }
+    {
+        auto service = node.service_builder(service_name).template publish_subscribe<uint32_t>().create().expect("");
+        auto static_config = service.static_config();
+        ASSERT_THAT(static_config.message_type_details().payload().type_name(), StrEq("u32"));
+    }
+    {
+        auto service = node.service_builder(service_name).template publish_subscribe<uint64_t>().create().expect("");
+        auto static_config = service.static_config();
+        ASSERT_THAT(static_config.message_type_details().payload().type_name(), StrEq("u64"));
+    }
+    {
+        auto service = node.service_builder(service_name).template publish_subscribe<int8_t>().create().expect("");
+        auto static_config = service.static_config();
+        ASSERT_THAT(static_config.message_type_details().payload().type_name(), StrEq("i8"));
+    }
+    {
+        auto service = node.service_builder(service_name).template publish_subscribe<int16_t>().create().expect("");
+        auto static_config = service.static_config();
+        ASSERT_THAT(static_config.message_type_details().payload().type_name(), StrEq("i16"));
+    }
+    {
+        auto service = node.service_builder(service_name).template publish_subscribe<int32_t>().create().expect("");
+        auto static_config = service.static_config();
+        ASSERT_THAT(static_config.message_type_details().payload().type_name(), StrEq("i32"));
+    }
+    {
+        auto service = node.service_builder(service_name).template publish_subscribe<int64_t>().create().expect("");
+        auto static_config = service.static_config();
+        ASSERT_THAT(static_config.message_type_details().payload().type_name(), StrEq("i64"));
+    }
+    {
+        auto service = node.service_builder(service_name).template publish_subscribe<bool>().create().expect("");
```

Comment:
For the `Slice<T>` tests further down, you also added `f32` and `f64`. In theory the same code path are used but just for completeness, could you also add them here?

Reply:
Ah sorry, I overlooked it.